DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,40 @@
+***
+Setup you dev environment.
+---------------------------
+If you are developing in Java and have maven installed, you can skip this setup and work with the work environment of your choice.
+However if you don't have an active Java development environment, please follow the following to make sure you are all set up and ready to write the code for the excercise.
+
+// todo - the purpese of this readme is to help setup a working environment in conferences where participants have almost no knowledge in Java development. 
+// todo - this is what we thoght might help in conferences like Hackeriot where the level of Java knowlegde is CS graduate level at most and bootcamp or personal interest knowledge is even more common.
+// todo - pass all text with chatgpt.
+// todo - write this as a trubleshoot guide - with how to test that the step was success full and perhaps guide how to fix common problems (path, restart, and others) 
+
+Prepare work environment with Maven and IDE 
+--------------------------------------------
+To prepare your work environment, follow these steps:
+1. IDE. \
+   If you don't have a preferred working IDE, download and install [Intellij](https://www.jetbrains.com/idea/download/) Community Edition for free. Make sure you download the Community Edition.
```

Comment:
The second sentence is superfelous - you already said download the Community Edition